src/Components/test/Ignitor/Ignitor.csproj

```diff
@@ -0,0 +1,16 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Reference Include="AngleSharp" />
```

Comment:
AngleSharp?? Is this because you're planning to parse markup frame content at some point?

Reply:
lmao, def not. Going to remove that